recipes/sensor.rb

```diff
@@ -0,0 +1,61 @@
+# install and run the metaflows sensor/collector
+include_recipe 'runit'
+
+if node['platform'] == 'centos'
```

Comment:
Right, so if someone runs this recipe on a non-CentOS server, it's a mistake. The way it's written now, Chef will skip everything in this block and claim it succeeded at the end. My opinion would be that the Chef run should fail.